src/lib.rs

```diff
@@ -334,11 +340,7 @@ where
         account_id: &<T as System>::AccountId,
         call: C,
     ) -> Result<SignedPayload<Encoded, <E as SignedExtra<T>>::Extra>, Error> {
-        let account_nonce = self
-            .fetch(AccountStore(account_id), None)
-            .await?
-            .unwrap()
-            .nonce;
+        let account_nonce = self.account(account_id).await?.unwrap().nonce;
```

Comment:
I know this was already there but I'd like the `unwrap()` replacing with `ok_or("Account not found")?`

Reply:
I don't know if that can actually ever happen. We need a test case for `Option<T>` and `T`. I think if the returns type is declared as `Option<T>` it should work out. Fetch always returns a default value. I'll remove this for now.